test/testsets/GlassCat.jl

```diff
@@ -117,69 +113,182 @@ using StaticArrays
+            "     B:                                            0.0044096",
+            "     C:                                            -2.85878e-5",
+            "     D:                                            0.0",
+            "     E:                                            0.0",
+            "     F:                                            0.0",
+            "Valid wavelengths:                                 0.32μm to 1.55μm",
+            "Reference temperature:                             25.0°C",
+            "Thermal ΔRI coefficients:",
+            "     D₀:                                           -0.0009083144750540808",
+            "     D₁:                                           0.0",
+            "     D₂:                                           0.0",
+            "     E₀:                                           0.0",
+            "     E₁:                                           0.0",
+            "     λₜₖ:                                          0.0",
+            "TCE (÷1e-6):                                       700.0",
+            "Ignore thermal expansion:                          false",
+            "Density (p):                                       0.878g/m³",
+            "ΔPgF:                                              0.008",
+            "RI at sodium D-Line (587nm):                       1.457587",
+            "Abbe Number:                                       57.19833",
+            "Cost relative to N_BK7:                            ?",
+            "Status:                                            Standard (0)",
+            "Melt frequency:                                    ?",
+            "Exclude substitution:                              false",
+            "Transmission data:",
+            "     Wavelength   Transmission      Thickness",
+            "         0.32μm           0.15         10.0mm",
+            "        0.365μm           0.12        100.0mm",
+            "       0.4047μm           0.42        100.0mm",
+            "         0.48μm           0.78        100.0mm",
+            "       0.4861μm           0.79        100.0mm",
+            "       0.5461μm           0.86        100.0mm",
+            "       0.5893μm            0.9        100.0mm",
+            "       0.6328μm           0.92        100.0mm",
+            "       0.6439μm            0.9        100.0mm",
+            "       0.6563μm           0.92        100.0mm",
+            "       0.6943μm           0.98        100.0mm",
+            "         0.84μm           0.99        100.0mm",
+            "      0.10648μm           0.61        100.0mm",
+            "         0.13μm           0.39        100.0mm",
+            "        0.155μm           0.11        100.0mm",
+            ""
+        ]
+        io = IOBuffer()
+        info(io, CARGILLE.OG0607)
+        @test String(take!(io)) === join(info_lines, '\n')
+
+        # TODO the rest of the string tests
```

Comment:
Should this TODO be addressed now?

Reply:
Testing `info` is a bit tedious because of all the different possibilities and the fact that it outputs to IO. It doesn't have any dependencies and is probably fine, so I think I'll defer it as a low priority item until another time.